src/main/resources/controllers-context.xml

```diff
@@ -3,20 +3,35 @@
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xmlns:mvc="http://www.springframework.org/schema/mvc"
        xmlns:context="http://www.springframework.org/schema/context"
+       xmlns:aop="http://www.springframework.org/schema/aop"
        xsi:schemaLocation="http://www.springframework.org/schema/beans
        http://www.springframework.org/schema/beans/spring-beans.xsd
        http://www.springframework.org/schema/mvc
        https://www.springframework.org/schema/mvc/spring-mvc.xsd
        http://www.springframework.org/schema/context
-       https://www.springframework.org/schema/context/spring-context.xsd" default-autowire="constructor">
+       https://www.springframework.org/schema/context/spring-context.xsd
+       http://www.springframework.org/schema/aop
+       https://www.springframework.org/schema/aop/spring-aop.xsd"
+       default-autowire="constructor">
+
+    <bean id="dogServiceAspect" class="com.example.dog.utils.DogServiceAspect" />
+
+    <aop:config>
+        <aop:aspect ref="dogServiceAspect">
+            <aop:around
+                    pointcut="@annotation(com.example.dog.utils.CustomTransactional)"
+                    method="addTransactionalSupport" arg-names="pjp,jp"/>
```

Comment:
Are these arg names for the pointcut or for the advice?

Reply:
For advice